tests/hiprtc/TestCommon.hh

```diff
@@ -22,6 +22,9 @@
 
 #include <hip/hiprtc.h>
 #include <hip/hip_runtime_api.h>
+// TODO: Remove this header when it is included by the
+//       hip_runtime_api.h header.
+#include <hip/spirv_hip_host_defines.h>
```

Comment:
Why include `hip_runtime_api.h` instead of `hip_runtime.h` ?

Reply:
This header is meant to be compilable in C++ mode and non-hipcc compilers and the `hip_runtime_api.h` is suited for that (statement from the header: "This file can be compiled with a standard compiler.").